{"uuid": "21f10d00-eec7-401f-b47b-2d08e52cb6e6", "children": ["5363a859-b735-401f-9747-db4cbcb8c709"], "befores": [{"name": "init_driver", "status": "broken", "statusDetails": {"message": "UnboundLocalError: local variable 'web_driver' referenced before assignment\n", "trace": "  File \"C:\\Users\\SauravSharma\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\SauravSharma\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\_pytest\\fixtures.py\", line 1111, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Users\\SauravSharma\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\_pytest\\fixtures.py\", line 883, in call_fixture_func\n    fixture_result = next(generator)\n  File \"C:\\Users\\SauravSharma\\Pytest\\Pom-Pytest\\Tests\\conftest.py\", line 12, in init_driver\n    request.cls.driver = web_driver\n"}, "start": 1648475879935, "stop": 1648475879935}], "start": 1648475879935, "stop": 1648475879955}